cytominer_database/ingest.py

```diff
@@ -69,7 +70,7 @@ def into(input, output, name, identifier, skip_table_prefix=False):
     """
 
     with backports.tempfile.TemporaryDirectory() as directory:
-        source = os.path.join(directory, os.path.basename(input))
+        source = os.path.join(os.path.abspath(directory), os.path.basename(input))
```

Comment:
`os.path.abspath` is probably harmless, but could you clarify why you're doing it? @gwaygenomics